CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file.
 - A bug in UVFlag where polarization array states were not updated when using `force_pol` keyword in `to_antenna` and `to_baseline`
 - A bug in UVFlag.to_baseline() where force_pol kwarg did not work for UVData Npols > 1
 - `UVData.read_uvfits` no longer breaks if there are non-ascii bytes in antenna names (which CASA sometimes writes).
+- A bug in `UVCal` objects that prevented them from properly getting data with `ee`/`nn`-style polarizations.
```

Comment:
This should be at the top of this list -- changelogs are in reverse chronological order.

Reply:
Fixed.